packages/application-components/src/components/public-page-layout/public-page-layout.tsx

```diff
@@ -41,6 +44,666 @@ const Container = styled.div`
   background-size: cover;
   background-position: center;
 `;
+const NewContainer = styled.div`
```

Comment:
It might make sense to call this something like `GradientBackground`, since it won't always be a `NewContainer`?